app/_src/using-mesh/managing-ingress-traffic/builtin-routes.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Configuring built-in routes
+---
+
+For configuring how traffic is forwarded from a listener to your mesh services,
+use [`MeshHTTPRoute`](/docs/{{ page.version }}/policies/meshhttproute) and
+[`MeshTCPRoute`](/docs/{{ page.version }}/policies/meshtcproute).
+
+Using these route resources with a gateway requires [using `spec.targetRef` to target
+gateway data plane proxies](../../../policies/targetref/#target-resources).
+Otherwise, [filtering and routing traffic](/../../../policies/meshhttproute) is
```

Comment:
this 404s, no idea why, but it generates https://deploy-preview-1582--kuma.netlify.app/policies/meshhttproute which is missing `docs/dev`. Can you use absolute links for this:

```suggestion
Otherwise, [filtering and routing traffic](/docs/{{ page.version }}/policies/meshhttproute) is
```